modules/gdscript/gdscript_compiler.h

```diff
@@ -139,6 +139,7 @@ class GDScriptCompiler {
 	bool within_await = false;
 
 public:
+	static void _convert_to_initializer_type(Variant &p_variant, const GDScriptParser::VariableNode *p_node);
```

Comment:
Since its public, we would usually not start the name with an underscore (which is the convention used for private methods).

Reply:
Yes, good call! Done! :)